lib/internal/modules/helpers.js

```diff
@@ -298,6 +298,30 @@ function normalizeReferrerURL(referrer) {
   return new URL(referrer).href;
 }
 
+/**
+ * Determines whether a specifier is a relative path.
+ * @param {string} specifier - The specifier to check.
+ */
+function isRelativeSpecifier(specifier) {
```

Comment:
Moving to a different file should be implemented in a different commit for future observability.

Reply:
Happy to do that, do folks typically do that as different PRs or multiple commits in the one PR?